doc/source/whatsnew/v0.19.1.txt

```diff
@@ -37,6 +37,7 @@ Bug Fixes
 - Bug in localizing an ambiguous timezone when a boolean is passed (:issue:`14402`)
 
 
+- Bug in ``Series.sort_index`` where parameters ``kind`` and ``na_position`` did not exist (:issue:`13589`, :issue:`14444`)
```

Comment:
@jorisvandenbossche want to highlite this change. maybe enhancements section? (even though its a bug fix / api change type of fix)